packages/webview_flutter/android/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,23 @@
-<manifest package="io.flutter.plugins.webviewflutter">
+<manifest package="io.flutter.plugins.webviewflutter"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <application>
```

Comment:
```suggestion

    <queries>
        <package android:name="com.google.android.apps.photos" />
        <package android:name="com.google.android.apps.docs" />
        <package android:name="com.google.android.documentsui" />
    </queries>

    <application>
```

https://developer.android.com/reference/android/content/pm/PackageManager

> Note: If your app targets Android 11 (API level 30) or higher, the methods in this class each return a filtered list of apps. Learn more about how to manage [package visibility](https://developer.android.com/training/package-visibility).

Reply:
Not sure if we should add this to the manifest of the web view. This is only relevant for users that want to support uploading a file from a web page (and only for API 30 and up). Wouldn't it be better to have the users add this to the manifests of their projects? We should then properly document that adding this is required for things to work of course...